_sass/_functions.scss

```
@@ -27,80 +31,101 @@ $modules: () !default;
 
 // RANGES
 // We use these functions to define ranges for various things, like media queries.
-@function lower-bound($range){
-  @if length($range) <= 0 {
+@function lower-bound($range) {
+  @if length($range) <=0 {
```

These changes are strange (in many places), do you have a code style where not both sides of operators have spaces?
```suggestion
  @if length($range) <= 0 {
```

---

Ahhh yeah I think this is done by default in the sass plugin I have, must have missed it, will try to fix that.